src/transformStateWithClones.js

```diff
@@ -7,7 +9,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
-}
+  const history = [];
+  let current = { ...state };
```

Comment:
Good job!
5. [cheklist]  [NAMING] - use proper names for object current

BAD EXAMPLE:

const current = { ...state }

GOOD EXAMPLE:

const currentState = { ...state }

Reply:
> Good job! 5. [cheklist] [NAMING] - use proper names for object current
> 
> BAD EXAMPLE:
> 
> const current = { ...state }
> 
> GOOD EXAMPLE:
> 
> const currentState = { ...state }

Good point